pkg/admission/interfaces.go

```diff
@@ -57,6 +59,16 @@ type Interface interface {
 	Handles(operation Operation) bool
 }
 
+// AdmissionPluginHost is interface between admission plugins and rest of
+// kube-apiserver.
+type AdmissionPluginHost interface {
```

Comment:
super nit picky: Is `AdmissionPluginHost` really the right name for this? Think think of anything better though.